Containerfile

```diff
@@ -12,10 +12,3 @@ ADD . /.
 
 RUN Rscript -e "pak::pkg_install('local::wweval')"
 RUN Rscript -e "pak::pkg_install('argparser')"
-
-RUN mkdir -p stanmodels
-RUN Rscript -e "cfaforecastrenewalww::compile_model( \
-  'cfaforecastrenewalww/inst/stan/renewal_ww_hosp_site_level_inf_dynamics.stan', \
-  'cfaforecastrenewalww/inst/stan', \
-  'stanmodels' \
-)"
```

Comment:
I tried to using ` system.file("stan", "wwinference.stan",package = "wwinference" )` but I think I might need the full relative file path (this gave me an error when I tried to build the container).

Reply:
What's the error you are getting?

Reply:
```
11/11: RUN Rscript -e "wwinference:compile_model(   'system.file("stan","wwinference.stan", package = "wwinference"),   'system.file("stan",package = "wwinference"),   'stanmodels' )"
.Rprofile for container loaded successfully
Error: unexpected symbol in "wwinference:compile_model(   'system.file(stan,wwinference.stan, package = wwinference),   'system.file"
Execution halted
Error: error building at STEP "RUN Rscript -e "wwinference:compile_model(   'system.file("stan","wwinference.stan", package = "wwinference"),   'system.file("stan",package = "wwinference"),   'stanmodels' )"": error while running runtime: exit status 1
```
I think it is because we can't have R scripting here, just need a file path?

Reply:
Change

```bash
11/11: RUN Rscript -e "wwinference:compile_model(   'system.file("stan","wwinference.stan", package = "wwinference"),   'system.file("stan",package = "wwinference"),   'stanmodels' )"
```

to 

```bash
11/11: RUN Rscript -e "wwinference:compile_model(   system.file('stan','wwinference.stan', package = 'wwinference'),   system.file('stan', package = 'wwinference'),   'stanmodels')"

```

I think... You have issues with the quotes, that's the problem.